server/rest/phase.py

```diff
@@ -106,6 +106,12 @@ def listPhases(self, challenge, params):
                dataType='boolean', default=False, required=False)
         .param('enableDocumentationUrl', 'Enable submission Documentation URL field.',
                dataType='boolean', default=False, required=False)
+        .param('requireOrganization', 'Require submission Organization field.', dataType='boolean',
+               default=True, required=False)
+        .param('requireOrganizationUrl', 'Require submission Organization URL field.',
+               dataType='boolean', default=True, required=False)
+        .param('requireDocumentationUrl', 'Require submission Documentation URL field.',
+               dataType='boolean', default=True, required=False)
```

Comment:
At some point we should convert COVALIC to use `autoDescribeRoute`, but not today.